src/google_map.js

```diff
@@ -718,11 +724,8 @@ export default class GoogleMap extends Component {
           const div = overlay.div;
           const overlayProjection = overlay.getProjection();
           if (div && overlayProjection) {
-            const bounds = map.getBounds();
-            const ne = bounds.getNorthEast();
-            const sw = bounds.getSouthWest();
             const ptx = overlayProjection.fromLatLngToDivPixel(
-              new maps.LatLng(ne.lat(), sw.lng())
+              overlayProjection.fromContainerPixelToLatLng({ x: 0, y: 0 })
```

Comment:
Could you elaborate why setting `{ x:0, y:0 }` fixes the animation problem?

Reply:
Good question, this is subtle. There are two ways of getting the lat-lng coordinates of the map's top-left corner.

1. Using `map.getBounds()` gives you the *final bounds*. That's because it is based on `map.getCenter()` and `map.getZoom()` which give you the final center and the final zoom. In other words, if you call `map.setZoom()` or click on the zoom control, `map.getZoom()` immediately reflects the new zoom, and so does `map.getBounds()`. You would use this if you were, say, fetching data from a server based on the map bounds.

2. Using `overlayProjection.fromContainerPixelToLatLng({ x: 0, y: 0})` gives you the bounds for *the current frame*. We use this because we are drawing each frame.

Reply:
@stephenfarrar Great, thanks for the answer. That makes sense, if `map.getBounds` gives you the _final bounds_, that is exactly what is making the markers animate from the corners.